tpch/dss.sql

```diff
@@ -1,6 +1,4 @@
 -- Sccsid:     @(#)dss.ddl	2.1.8.1
-CREATE DATABASE IF NOT EXISTS TPCH;
```

Comment:
Better to keep it.

Reply:
In our environment, we often have more than one TPC-H dataset. Use `TPC-H$scale_factor` is better.